staff/groups/aleti/github-app-2.0.0-react/src/App.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from 'react';
+import SearchUsers from './components/SearchUsers'
+import UsersList from './components/UsersList'
+import UserDetails from './components/UserDetails'
+import logic from './logic/index'
+
+import './App.css';
+
+class App extends Component {
+  state = {
+    users: [],
+    user: {},
+    query: '',
```

Comment:
WHY query in App state, instead of SearchUsers state? query in this case is responsibility of SearchUsers, and later on, when form submitted inside, the component itself can call its props searchMethod received from parent, passing the query as a parameter. CHECK this demo to understand the idea => https://github.com/manuelbarzi/skylab-bootcamp-201804/blob/develop/stuff/github-app-react-2.1.0/src/components/Search.js

Reply:
+1